src/sparseml/pytorch/sparsification/quantization/quantize_qat_export.py

```diff
@@ -666,6 +666,237 @@ def _convert_quantizable_matmul(model: ModelProto):
         )
 
 
+def _convert_quantizable_matmul_matmul(model: ModelProto):
+    """
+    A pass for converting a MatMul into a quantized representation
+    This MatMul is the result of quantizing native torch.matmul using QATMatMul
+
+    | Starting with:
+    |          INPUT_0           INPUT_1
+    |            |               |
+    |     QuantizeLinear     QuantizeLinear
+    |            |               |
+    |     DequantizeLinear   DequantizeLinear
+    |                  |      |
+    |                   MatMul
+    |                     |
+    |                  Transpose       INPUT_2 (with constant kernel)
+    |                     |                      |
+    |                  Reshape           QuantizeLinear
+    |                     |                   |
+    |               QuantizeLinear      DequantizeLinear
+    |                       |                |
+    |                DequantizeLinear  Transpose
+    |                            |     |
+    |                            MatMul
+    |                              |
+    |                             Add   (with constant bias)
+    |                              |
+    |                           OUTPUT
+    |
+    | We end up converting to:
+    |          INPUT_0           INPUT_1
+    |            |               |
+    |     QuantizeLinear     QuantizeLinear
+    |                  |      |
+    |                 QLinearMatMul
+    |                     |
+    |                 Transpose
+    |                     |
+    |                  Reshape
+    |                     |
+    |              MatMulInteger (with constant uint8 kernel)
+    |                    |
+    |                  Add (constant bias + zero point correction)
+    |                   |
+    |                  Cast (INT32 -> FP32)
+    |                   |
+    |                  Mul (Rescale from bias scale)
+    |                  |
+    |                OUTPUT
+    """
+    conversion_count = 0
+    matmul_nodes = [n for n in model.graph.node if n.op_type in ["MatMul"]]
+    for matmul_node in matmul_nodes:
+        graph = ONNXGraph(model)
```

Comment:
could make this graph outside of the loop and regenerate it at the end of the loop if we did the conversion